source/core/parameters_lagrangian.cc

```diff
@@ -533,12 +538,24 @@ namespace Parameters
                           "0.0, 0.0, 0.0",
                           Patterns::List(Patterns::Double()),
                           "Initial angular velocity (x, y, z)");
+        auto initial_temperature_function_parsed =
+          std::make_shared<Functions::ParsedFunction<dim>>(1);
+        prm.enter_subsection("initial temperature function");
+        {
+#if DEAL_II_VERSION_GTE(9, 7, 0)
+          initial_temperature_function_parsed->declare_parameters(prm, 1, "0");
+#else
+          initial_temperature_function_parsed->declare_parameters(prm, 1);
+#endif
```

Comment:
```suggestion
          initial_temperature_function_parsed->declare_parameters(prm, 1);
```

You don't need to check the version if you are using "0" . It is the default value.

Reply:
@OGaboriault about that, I tried removing it and there was an error with the clang tidy so I put it back